lib/ReactViewModels/ViewState.js

```diff
@@ -159,6 +163,10 @@ export default class ViewState {
         this.activeTabCategory = 'my-data';
     }
 
+    setPanelToUserData(){
```

Comment:
I think it'd be ok to just call `openUserData`. When you know the explorer panel is already open, setting `explorerPanelIsVisible` to true again won't do any harm.